server/pse/pse_freebsd.go

```diff
@@ -54,7 +56,7 @@ int getusage(double *pcpu, unsigned int *rss, unsigned int *vss)
 
     *rss = pagetok(kp.ki_rssize);
     *vss = kp.ki_size;
-    *pcpu = kp.ki_pctcpu;
+    *pcpu = (double)kp.ki_pctcpu / FSCALE;
```

Comment:
IIRC this number can go up to 100% X num cores, so if you have 8 cores you can see numbers like 640%. Not sure if that is what is being done here.

Reply:
The data stored is not a floating point, it's a scaled integer.  `FSCALE` is the amount it was multiplied by to get an integer.  We need to divide by `FSCALE` when restoring to a float.

Reply:
FYI, this same calculation but in the Golang amd64 version, `f_pcpu := float64(r_pcpu) / float64(2048)`, yielded "reasonable" values for me in testing just now.

While running `nats bench foo` in one tty, `nats server request variables` showed `.data.cpu` rise up as high as `0.24658203125`.